addons/account/controllers/portal.py

```diff
@@ -29,6 +31,11 @@ def _prepare_home_portal_values(self, counters):
     # My Invoices
     # ------------------------------------------------------------
 
+    def _get_overdue_invoice_count(self):
+        overdue_invoice_count = request.env['account.move'].search_count(self._get_overdue_invoices_domain()) \
```

Comment:
This is not feasible in this case, I would have to use if condition instead of ternary operator because here if no access rights granted, I should not attempt to search for overdue moves.